src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeInterfaces.hpp

```diff
@@ -160,8 +159,13 @@ struct InitializeInterfaces {
                                      domain::Tags::InterfaceMesh<dim>>,
       tmpl::transform<SliceTagsToFace,
                       make_slice_tag<tmpl::_1, tmpl::pin<Directions>>>,
-      domain::Tags::InterfaceCompute<
-          Directions, domain::Tags::UnnormalizedFaceNormalCompute<dim>>,
+      tmpl::conditional_t<
+          UseMovingMesh,
+          tmpl::list<domain::Tags::InterfaceCompute<
+              Directions,
+              domain::Tags::UnnormalizedFaceNormalMovingMeshCompute<dim>>>,
+          domain::Tags::InterfaceCompute<
+              Directions, domain::Tags::UnnormalizedFaceNormalCompute<dim>>>,
```

Comment:
For my understanding: could the `conditional_t` be located inside the InterfaceCompute? like this:
```
domain::Tags::InterfaceCompute<Directions, tmpl::conditional_t<UseMovingMesh, ..., ...>>
```

I feel this would be clearer (puts the "if" around the type that is really changing), but perhaps there is a reason this cannot be done?